app/src/main/res/layout/fragment_search.xml

```diff
@@ -1,59 +1,76 @@
             android:orientation="vertical">
 
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/go_out"
-                android:gravity="center_horizontal"
-                android:textColor="@color/black"
-                android:layout_marginTop="@dimen/layout_margin_large"
-                android:textSize="@dimen/heading_text_size" />
-
-            <EditText
-                android:id="@+id/location_editText"
+            <LinearLayout
+                android:id="@+id/search_linear_layout"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:hint="@string/anywhere"
-                android:layout_marginTop="@dimen/layout_margin_large"
-                android:textSize="@dimen/heading_text_size" />
-        </LinearLayout>
+                android:layout_margin="@dimen/layout_margin_extra_large"
+                android:gravity="center"
+                android:orientation="vertical">
 
-        <FrameLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent">
-
-            <ProgressBar
-                android:layout_gravity="center"
-                android:id="@+id/progressBar"
-                android:visibility="gone"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content" >
-            </ProgressBar>
-
-            <android.support.v7.widget.RecyclerView
-                android:id="@+id/eventsRecycler"
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/layout_margin_large"
+                    android:gravity="center_horizontal"
+                    android:text="@string/go_out"
+                    android:textColor="@color/black"
+                    android:textSize="@dimen/heading_text_size" />
+
+                <EditText
+                    android:id="@+id/location_editText"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/layout_margin_large"
+                    android:hint="@string/anywhere"
+                    android:textSize="@dimen/heading_text_size" />
+            </LinearLayout>
+
+            <FrameLayout
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:scrollbars="vertical" />
-        </FrameLayout>
-    </LinearLayout>
-</android.support.v4.widget.NestedScrollView>
\ No newline at end of file
+                android:layout_height="match_parent">
+
+                <ProgressBar
+                    android:id="@+id/progressBar"
```

Comment:
snake case

Reply:
view ids should remain to be camel case as they are picked up by kotlinx